### DISCUSSION TITLE
2024/02/10/acm/codeforces/CodeforcesRound912/

### DISCUSSION THREAD

**Discussion by giscus** [Hukeqing/Hukeqing.github.io#12] [General(讨论贴)]
# 2024/02/10/acm/codeforces/CodeforcesRound912/

Codeforces Round 912 (Div. 2) 个人写题记录

https://blog.mauve.icu/2024/02/10/acm/codeforces/CodeforcesRound912/

**Comment by kernelbin** [comment]
带带我带带我！
